static/assets/js/index.js

```diff
@@ -9,12 +12,37 @@
     });
     $('#search-field').keyup(function(e) {
       if (e.keyCode == 27) {
-      $('#js-search-focus').focus();
-  }
-});
+        $('#js-search-focus').focus();
+      }
+    });
+    $govBarWrapper.on('click', function(){
+      toggleGovBar();
+    })
     ajaxifyContactForm();
   });
 
+  function toggleGovBar() {
+    if ($('#gov-banner').hasClass('hide')) {
```

Comment:
You can use the `$govBarContent` variable here I think.

Reply:
Updated.